x-pack/plugins/secops/common/graphql/types.ts

```diff
@@ -0,0 +1,43 @@
+/* tslint:disable */
+import { GraphQLResolveInfo } from 'graphql';
```

Comment:
To get the Elastic license at the top of this file you will open up your

`gpl_gen.json`

And add these lines:

```json
  "generatorConfig": {
    "prepend": [
      "/*",
      " * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one",
      " * or more contributor license agreements. Licensed under the Elastic License;",
      " * you may not use this file except in compliance with the Elastic License.",
      " */"
    ]
  },
```

Full file looks like:
```json
{
  "flattenTypes": true,
  "generatorConfig": {
    "prepend": [
      "/*",
      " * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one",
      " * or more contributor license agreements. Licensed under the Elastic License;",
      " * you may not use this file except in compliance with the Elastic License.",
      " */"
    ]
  },
  "primitives": {
    "String": "string",
    "Int": "number",
    "Float": "number",
    "Boolean": "boolean",
    "ID": "string"
  }
}
```

ref: https://www.npmjs.com/package/graphql-codegen-typescript-template#prepend--default-value-null-


However that only works with the latest generator:

```sh
"graphql-code-generator": "^0.12.6",
"graphql-codegen-introspection-template": "^0.12.6",
"graphql-codegen-typescript-template": "^0.12.6",
```

And we are on version:

```sh
"graphql-code-generator": "^0.10.1",
"graphql-codegen-introspection-template": "^0.10.5",
"graphql-codegen-typescript-template": "^0.10.1",
```

However, it does not blow up and is forward compatible in my testing swapping between versions, so still recommend it.